### PR TITLE
Feat: New Parser for 'dotnet --version' Command for Containers

### PR DESCRIPTION
**Pull request by TZ3070**
Signed-off-by: Xinting Li <xintli@redhat.com>

### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### Complete Description of Additions/Changes:

New parser for dotnet --version in container

### CODE REVIEW

File: insights/parsers/dotnet.py
Comment (xiangce) [inline insights/parsers/dotnet.py:57]: ContainerDotNetVersion

**Review by xiangce — CHANGES_REQUESTED** [review]
And please update the git commit msg more relevant.

**Review by xiangce — APPROVED** [review]
Approved pending the approval of the Spec

**Review by psachin — APPROVED** [review]

**Pull request merged**
